analysis/deathknight/src/shadowlands/covenants/DeathsDue.tsx

```diff
@@ -221,7 +231,7 @@ class DeathsDue extends Analyzer {
         .recommended(`${recommended} recommended`),
     );
   }
-  statistic() {
+  statistic(): JSX.Element {
```

Comment:
adding types to overridden methods isn't necessary (but you don't need to go back and remove them from this PR)

Reply:
Yea, I don't have any better explanation than "I felt it needed type annotation", despite already being correctly suggested 😄 Will get rid of it in places where it's not needed.